action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
     description: "Optionally specify a specific tag from the Webots repository to install. (Defaults to webotsVersion)"
     required: false
     default: ''
+  webotsBaseUrl:
+    description: "Optionally specify a base URL containing the distribution files (Defaults to https://github.com/cyberbotics/webots/releases/download/<webotsTag>)"
+    required: false
+    default: ''
```

Comment:
```suggestion
    default: 'https://github.com/cyberbotics/webots/releases/download/'
```

Reply:
I think we want that to remain empty so that the code detects it's absence. Similar to webotsTag.

Reply:
Ah, my bad. I read the docs wrong.